README.md

```diff
@@ -2,6 +2,15 @@
 
 `image-tools` is a collection of tools for working with the [OCI image format specification](https://github.com/opencontainers/image-spec).
 
+# Building
+
+```
+go get github.com/opencontainers/image-tools/
+cd $GOPATH/src/github.com/opencontainers/image-tools/
+make tools
+cp oci-unpack oci-create-runtime-bundle oci-image-validate /usr/local/bin
```

Comment:
We've been using `$ ` prompts for shell-session examples (e.g. [here][1]), otherwise this looks good to me.

Longer term, we may want an installation target like [this][2].

[1]: https://github.com/opencontainers/image-tools/blob/421458f7e467ac86175408693a07da6d29817bf7/cmd/oci-unpack/oci-unpack.1.md#examples
[2]: https://github.com/opencontainers/runtime-tools/blob/893d9e32f4088f4928eac165ee3e38b312ea9582/Makefile#L18